python-sdk/src/astro/databricks/api_utils.py

```diff
@@ -18,6 +21,49 @@
 log = logging.getLogger(__name__)
 
 
+def delete_secret_scope(scope_name: str, api_client: ApiClient) -> None:
+    """
+    Delete the scope we created to prevent littering the databricks secret store with one-time scopes.
+
+    :param scope_name: name of scope to delete
+    :param api_client: populated databricks client
+    """
+    secrets = SecretApi(api_client)
+    try:
+        secrets.delete_scope(scope_name)
+    except HTTPError as h:
+        # We don't care if the resource doesn't exist since we're trying to delete it
+        if not h.response.json().get("error_code", "") == "RESOURCE_DOES_NOT_EXIST":
+            raise h
+
+
+def create_secrets(scope_name: str, filesystem_secrets: Dict[str, str], api_client: ApiClient) -> None:
```

Comment:
Is this the only way we support users loading from S3 into Datbricks?
Are we too opinionated here? Would it be a possibility for the user to manage their secrets and just give us the key?

Reply:
@tatiana this is how we can ensure that users can use connections that exist in the Airflow DB for loading to databricks. We can make an "assume_secrets_exist" mode for users who want to pre-load these values into their databricks cluster, but I think that would be a good follow-up ticket.

Reply:
@tatiana I've added the ability to turn off secret loading and made it a global environment variable. I've also added documentation showing the two options so users can decide for themselves.